management/server/peer/peer.go

```diff
@@ -64,21 +64,48 @@ type Location struct {
 	GeoNameID    uint // city level geoname id
 }
 
+// NetworkAddress is the IP address with network and MAC address of a network interface
+type NetworkAddress struct {
+	NetIP string
```

Comment:
shouldn't this be netip.Prefix?

Reply:
This is the server code. If I remember well you suggested that, I should not touch now the server side.

Reply:
I see, probably was my mistake. We will need to update it as well, otherwise, we will need to convert from string to netip.Prefix later